### PR TITLE
first try solution

### PR DESCRIPTION
**Pull request by AndreyDMA**

### CODE REVIEW

File: src/main/java/core/basesyntax/FlipBit.java [hunk 1]
```diff
@@ -6,10 +6,37 @@
...
+            array[bitIndex - 1] = '1';
+        }
+        for (int i = 0; i < array.length; i++) {
+            number += Character.getNumericValue(array[i]) * Math.pow(2, i);
+        }
+        return number;
     }
-}
+
+}
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FlipBit.java]: add a blank line

File: src/main/java/core/basesyntax/FlipBit.java [hunk 2]
```diff
@@ -6,10 +6,37 @@
      * Реализуйте метод flipBit(int value, int bitIndex), изменяющий значение одного бита
      * заданного целого числа на противоположное.
      * Данная задача актуальна, например, при работе с битовыми полями.
-     *
      * Договоримся, что биты нумеруются от младшего (индекс 1) к старшему (индекс 32).
      */
     public int flipBit(int value, int bitIndex) {
-        return 0;
+        String binaryNumber = "";
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FlipBit.java]: original solution)))
try to simplify and operate with binary system

File: src/main/java/core/basesyntax/FlipBit.java [hunk 3]
```diff
@@ -6,10 +6,27 @@
      * Реализуйте метод flipBit(int value, int bitIndex), изменяющий значение одного бита
      * заданного целого числа на противоположное.
      * Данная задача актуальна, например, при работе с битовыми полями.
-     *
      * Договоримся, что биты нумеруются от младшего (индекс 1) к старшему (индекс 32).
      */
     public int flipBit(int value, int bitIndex) {
-        return 0;
+        String binaryNumber = Integer.toBinaryString(value);
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FlipBit.java]: try to use << and ^ operators

Comment (AndreyDMA) [reply src/main/java/core/basesyntax/FlipBit.java]: done))

**Pull request closed without merging**